### PR TITLE
Support MDC workload

### PR DESCRIPTION
**Pull request by MSRonBorysowski**

### CODE REVIEW

File: api-reference/beta/resources/security-googlecloudresourceevidence.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Namespace: microsoft.graph.security
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents Google cloud resource such as compute or Kubernetes cluster identifier.
+
+Inherits from [alertEvidence](../resources/security-alertevidence.md).
+
+## Properties
```
Comment (JarbasHorst) [inline api-reference/beta/resources/security-googlecloudresourceevidence.md:20]: Please, add to this Properties table in alphabetical order the properties that are inherited from "alertEvidence".

Please, apply this fix also here: 
- api-reference/v1.0/resources/security-googlecloudresourceevidence.md

File: api-reference/beta/resources/security-alert.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ When detecting a threat, a security provider creates an alert in the system. Mic
...
 |resolvedDateTime|DateTimeOffset|Time when the alert was resolved.|
-|serviceSource|[microsoft.graph.security.serviceSource](#servicesource-values)|The service or product that created this alert. Possible values are: `microsoftDefenderForEndpoint`, `microsoftDefenderForIdentity`, `microsoftCloudAppSecurity`, `microsoftDefenderForOffice365`, `microsoft365Defender`, `aadIdentityProtection`, `appGovernance`, `dataLossPrevention`.|
+|serviceSource|[microsoft.graph.security.serviceSource](#servicesource-values)|The service or product that created this alert. Possible values are: `microsoftDefenderForEndpoint`, `microsoftDefenderForIdentity`, `microsoftCloudAppSecurity`, `microsoftDefenderForOffice365`, `microsoft365Defender`, `aadIdentityProtection`, `appGovernance`, `dataLossPrevention`, `unknownFutureValue`, `microsoftDefenderForCloud`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value(s) in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `microsoftDefenderForCloud`.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/security-alert.md:56]: The serviceSource property doesn't list all supported enum members correctly. 

- `unknown` is missing
- should `microsoftCloudAppSecurity` be `microsoftDefenderForCloudApps`
- should `aadIdentityProtection` be `azureAdIdentityProtection`
- should `appGovernance` be `microsoftAppGovernance`

Please, apply the same fixes also here:
- api-reference/v1.0/resources/security-alert.md

File: api-reference/beta/resources/security-googlecloudresourceevidence.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+|Property|Type|Description|
+|:---|:---|:---|
+|location|String|The zone or region where the resource is located.|
+|locationType|[microsoft.graph.security.googleCloudLocationType](#locationtype-values)|The type of location. Possible values are: `unknown`, `regional`, `zonal`, `global`, `unknownFutureValue`.|
+|projectId|String|The Google project ID as defined by the user.|
+|projectNumber|Int64|The project number assigned by Google.|
+|resourceName|String|The name of the resource.|
+|resourceType|String|The type of the resource.|
+
+### locationType values
```
Comment (JarbasHorst) [inline api-reference/beta/resources/security-googlecloudresourceevidence.md:31]: What is the correct name for this enum: googleCloudLocationType or locationType?

If a fix is required, please apply this change also here:
- api-reference/v1.0/resources/security-googlecloudresourceevidence.md

File: api-reference/beta/resources/security-googlecloudresourceevidence.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+Represents a Google Cloud resource such as compute or Kubernetes cluster identifier.
+
+Inherits from [alertEvidence](../resources/security-alertevidence.md).
+
+## Properties
+
+|Property|Type|Description|
+|:---|:---|:---|
+|location|String|The zone or region where the resource is located.|
+|locationType|[microsoft.graph.security.googleCloudLocationType](#locationtype-values)|The type of location. Possible values are: `unknown`, `regional`, `zonal`, `global`, `unknownFutureValue`.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/security-googlecloudresourceevidence.md:25]: What is the correct name for this type: googleCloudLocationType or locationType? API Doctor throws the following exception: Referenced type microsoft.graph.security.googleCloudLocationType is not defined in the doc set! Potential suggestion: microsoft.graph.location

If a fix is required, please apply this change also here:
- api-reference/v1.0/resources/security-googlecloudresourceevidence.md

File: api-reference/beta/resources/security-amazonresourceevidence.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Namespace: microsoft.graph.security
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents an amazon resources that is reported as part of the security detection alert.
+
+Inherits from [alertEvidence](../resources/security-alertevidence.md).
+
+## Properties
```
[the remark this answers is not in the document]
Comment (MSRonBorysowski) [reply api-reference/beta/resources/security-amazonresourceevidence.md:20]: Not sure I understand this... we mention that it inherits from the base class... why should we add the properties here as well? what if someday we add `alertEvidence` (base class) property values? should we change all the sub classes as well?

[the remark this answers is not in the document]
Comment (Jarbas-MSFT) [reply api-reference/beta/resources/security-amazonresourceevidence.md:20]: This represents the consistency bar for Microsoft Graph docs. The properties/relationships of a base class should also be listed in the resource(s) that inherits from it. This helps readers quickly understand what properties/relationships are available for a resource. In interest of the time, I added the missing properties to the resource topics.

**Comment by JarbasHorst** [comment]
Hi @Lauragra, this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**